source/tyr/tyr/api.py

```diff
@@ -78,6 +78,8 @@
 api.add_resource(resources.AutocompleteUpdateData,
                  '/v0/autocomplete_parameters/<ac_instance_name>/update_data')
 
+api.add_resource(resources.MigrateFromPoiToOsm, '/v0/instances/<string:instance_name>/:actions/migrate_from_poi_to_osm')
```

Comment:
consistency: In tartare URL is `/actions/<my_action>` https://github.com/CanalTP/tartare/blob/master/documentation/ressources/open_api.yaml#L182

Reply:
changed to '/v0/instances/<string:instance_name>/actions/migrate_from_poi_to_osm'